web/app/[workspaceSlug]/(projects)/profile/[userId]/header.tsx

```diff
@@ -6,25 +6,30 @@ import { observer } from "mobx-react";
 import Link from "next/link";
 import { useParams } from "next/navigation";
 import { ChevronDown, PanelRight } from "lucide-react";
+import { IUserProfileProjectSegregation } from "@plane/types";
 import { Breadcrumbs, CustomMenu } from "@plane/ui";
 import { BreadcrumbLink } from "@/components/common";
 // components
+import { ProfileIssuesFilter } from "@/components/profile";
 import { PROFILE_ADMINS_TAB, PROFILE_VIEWER_TAB } from "@/constants/profile";
 import { cn } from "@/helpers/common.helper";
 import { useAppTheme, useUser } from "@/hooks/store";
 
 type TUserProfileHeader = {
+  userProjectsData: IUserProfileProjectSegregation | undefined;
   type?: string | undefined;
+  showProfileIssuesFilter?: boolean;
 };
 
 export const UserProfileHeader: FC<TUserProfileHeader> = observer((props) => {
-  const { type = undefined } = props;
+  const { userProjectsData, type = undefined, showProfileIssuesFilter } = props;
   // router
   const { workspaceSlug, userId } = useParams();
   // store hooks
   const { toggleProfileSidebar, profileSidebarCollapsed } = useAppTheme();
   const {
     membership: { currentWorkspaceRole },
+    data: currentUser,
   } = useUser();
   // derived values
   const AUTHORIZED_ROLES = [20, 15, 10];
```

Comment:
Why is this not in the global constants file?

Reply:
Resolved